### DISCUSSION TITLE
Google Summer of Code 2022

### DISCUSSION THREAD

**Discussion by patel-zeel** [cornellius-gp/gpytorch#1921] [Q&A]
Hi,

I was wondering if anyone from the GPyTorch team is releasing some projects in GSOC 2022. I would love to work on several useful projects similar to [these](https://turing.ml/dev/posts/2022-02-17-gsoc).

**Comment by gpleiss** [comment]
It looks like we're a bit too late to apply as an organization... 😕

Thanks for bringing this to our attention. We can look into applying for next year!
